go.mod

```diff
@@ -0,0 +1,21 @@
+module github.com/gutakk/go-google-scraper
+
+go 1.15
+
+require (
+	github.com/gin-gonic/gin v1.6.3
+	github.com/go-playground/validator/v10 v10.4.1 // indirect
```

Comment:
Not sure about many of these dependencies (this one for playground and others like `protobuf`) 🤔 Might be good to only keep what you plan to use.

This [boilerplate repository](https://github.com/Massad/gin-boilerplate/blob/master/go.mod) does not have many of these dependencies for instance

Reply:
Agree with you. I setup gin following [Gin Gonic repo](https://github.com/gin-gonic/gin) and they suggest to use `go get -u github.com/gin-gonic/gin`

Removed unused dependencies in f1f3204c58bf3112b3851cbcfe420748f75a0169